src/main/java/com/j256/simplemagic/entries/IanaEntries.java

```diff
@@ -0,0 +1,112 @@
+package com.j256.simplemagic.entries;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.zip.GZIPInputStream;
+
+/**
+ * Loads the IANA databases (build on 10 august 2017)
+ * @author Jean-Christophe Malapert (jcmalapert@gmail.com)
+ */
+public class IanaEntries {
+
+    private final static String INTERNAL_IANA_APP_DB = "/iana_app.gz";
+    private final static String INTERNAL_IANA_AUDIO_DB = "/iana_audio.gz";
+    private final static String INTERNAL_IANA_FONT_DB = "/iana_font.gz";
+    
+    /**
+     * The database
+     */
+    private final Map<String, IanaEntry> _ianaDB;
+
+    /**
+     * Constructor.
+     */
+    public IanaEntries() {
+        this._ianaDB = new HashMap<String, IanaEntry>();
+        loadDb(INTERNAL_IANA_APP_DB);
+        loadDb(INTERNAL_IANA_AUDIO_DB);
+        loadDb(INTERNAL_IANA_FONT_DB);
+    }
+
+    /**
+     * Loads the IANA database
+     * @param db 
+     */
+    private void loadDb(String db) {        
+        InputStream stream = getClass().getResourceAsStream(db);
+        if (stream == null) {
+            throw new RuntimeException(db + " is missing");
+        }
+        Reader reader = null;
+        try {
+            reader = new InputStreamReader(new GZIPInputStream(new BufferedInputStream(stream)));
+            BufferedReader lineReader = new BufferedReader(reader);
+            String line = "";
+            lineReader.readLine();//skip the first line
+            while ((line = lineReader.readLine()) != null) {
+                String[] ianaEntryParsed = line.split(","); 
+                if(ianaEntryParsed[2].startsWith("\"")) {
```

Comment:
This will get an out of bounds exception on a malformed line.  Maybe add a check to make sure that the ianaEntryParsed has 3 or more entries?